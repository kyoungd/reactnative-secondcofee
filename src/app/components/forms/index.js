import FormHeading from './FormHeading'
import SignInForm from './SignInForm'
import SignUpForm from './SignUpForm'
import ResetPasswordForm from './ResetPasswordForm'
import PersonalInformationForm from './PersonalInformationForm'
import ProfessionalExperienceForm from './ProfessionalExperienceForm'

export {
  FormHeading,
  SignInForm,
  SignUpForm,
  ResetPasswordForm,
  PersonalInformationForm,
  ProfessionalExperienceForm
}
