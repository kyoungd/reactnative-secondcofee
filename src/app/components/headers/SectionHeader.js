import React from 'react'
import PropTypes from 'prop-types'
import styled from 'styled-components/native'

const Wrapper = styled.Text`
  padding: 16px;
  font-family: 'SourceSansProBold';
  font-size: 13;
  color: #000;
  text-transform: uppercase;
  background-color: #f4f6fa;
`

export default function SectionHeader({ children }) {
  return <Wrapper>{children}</Wrapper>
}

SectionHeader.propTypes = {
  children: PropTypes.node.isRequired
}
