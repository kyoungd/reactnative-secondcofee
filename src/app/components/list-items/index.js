import FilterOptionListItem from './FilterOptionListItem'
import JobListItem from './JobListItem'
import ProfileDetailListItem from './ProfileDetailListItem'
import SavedJobListItem from './SavedJobListItem'
import SettingsListItem from './SettingsListItem'

export {
  FilterOptionListItem,
  JobListItem,
  ProfileDetailListItem,
  SavedJobListItem,
  SettingsListItem
}
