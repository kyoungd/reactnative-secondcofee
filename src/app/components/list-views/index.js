import FilterOptionsListView from './FilterOptionsListView'
import JobsListView from './JobsListView'
import SavedJobsListView from './SavedJobsListView'

export { FilterOptionsListView, JobsListView, SavedJobsListView }
