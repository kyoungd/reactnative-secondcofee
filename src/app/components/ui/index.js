import CustomActivityIndicator from './CustomActivityIndicator'
import CustomStatusBar from './CustomStatusBar'
import FormField from './FormField'
import RegularButton from './RegularButton'
import ActionButton from './ActionButton'
import SafeAreaViewWrapper from './SafeAreaViewWrapper'
import Separator from './Separator'
import ListEmptyComponent from './ListEmptyComponent'
import ListLoadingComponent from './ListLoadingComponent'
import MultiSliderComponent from './MultiSliderComponent'
import ImageProgressComponent from './ImageProgressComponent'
import ImageOverlayComponent from './ImageOverlayComponent'

export {
  CustomActivityIndicator,
  CustomStatusBar,
  FormField,
  RegularButton,
  ActionButton,
  SafeAreaViewWrapper,
  Separator,
  ListEmptyComponent,
  ListLoadingComponent,
  MultiSliderComponent,
  ImageProgressComponent,
  ImageOverlayComponent
}
