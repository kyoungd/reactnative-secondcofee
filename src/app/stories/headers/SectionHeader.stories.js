import React from 'react'
import { storiesOf } from '@storybook/react-native'
import { SectionHeader } from 'components/headers'
import { CenterView } from '../decorators'

storiesOf('SectionHeader', module)
  .addDecorator(story => <CenterView>{story()}</CenterView>)
  .add('default', () => <SectionHeader>HEADING</SectionHeader>)
