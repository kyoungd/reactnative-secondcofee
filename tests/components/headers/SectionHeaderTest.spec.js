import React from 'react'
import { shallow } from 'react-native-testing-library'
import { SectionHeader } from 'components/headers'

test('Component renders correctly', () => {
  const { output } = shallow(<SectionHeader>Heading</SectionHeader>)
  expect(output).toMatchSnapshot()
})
