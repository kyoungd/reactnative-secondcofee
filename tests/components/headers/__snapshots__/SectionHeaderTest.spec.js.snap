// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<ForwardRef>
  Heading
</ForwardRef>
`;
